From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Sat, 8 Aug 2026 18:33:55 +0200
Subject: [PATCH] com.webos.service.camera2: Fix outbound permission error to
 com.webos.app.camera

Fixes:

Apr 19 12:52:45 qemux86-64 ls-hubd[96]: [] [pmlog] ls-hubd LSHUB_NO_NAME_PERMS {} Can not find match for 'com.webos.app.camera-718' in pattern queue '["org.webosports.service.pdm", "org.webosports.service.notification", "org.webosports.pdm", "org.webosports.notification", "org.webosinternals.service.pdm", "org.webosinternals.service.notification", "org.webosinternals.pdm", "org.webosinternals.notification", "com.webos.service.pdm", "com.webos.notification", "com.palm.service.pdm", "com.palm.service.notification", "com.palm.pdm", "com.palm.notification", "com.lge.pdm", "com.lge.notification"]'
Apr 19 12:52:45 qemux86-64 ls-hubd[96]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.app.camera-718","SRC_APP_ID":"com.webos.service.camera2","EXE":"/usr/sbin/com.webos.service.camera2","PID":772} "com.webos.service.camera2" does not have sufficient outbound permissions to communicate with "com.webos.app.camera-718" (cmdline: /usr/sbin/com.webos.service.camera2)
Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending
---
 files/sysbus/com.webos.service.camera2.role.json.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.camera2.role.json.in b/files/sysbus/com.webos.service.camera2.role.json.in
index 1bd7713..ffc3326 100644
--- a/files/sysbus/com.webos.service.camera2.role.json.in
+++ b/files/sysbus/com.webos.service.camera2.role.json.in
@@ -13,7 +13,8 @@
             "outbound": [
                 "com.webos.service.pdm",
                 "com.webos.service.appcasting",
-                "com.webos.service.audio"
+                "com.webos.service.audio",
+                "com.webos.app.camera-*"
             ]
         },
         {
